migration/sql-files/002-fix-273-workitemtype.sql

```diff
@@ -0,0 +1,2 @@
+UPDATE work_item_types
+SET fields = '{"system.state": {"Type": {"Kind": "enum", "Values": ["new", "in progress", "resolved", "closed"], "BaseType": {"Kind": "string"}}, "Required": true}, "system.title": {"Type": {"Kind": "string"}, "Required": true}, "system.creator": {"Type": {"Kind": "user"}, "Required": true}, "system.assignee": {"Type": {"Kind": "user"}, "Required": false}, "system.description": {"Type": {"Kind": "string"}, "Required": false}}';
```

Comment:
right... but.....

1. This work item types update doesn't contain the 'remote_item_id' nor the new 'open' state.
2. Shouldn't this be part of PopulateCommon part and not the base database structure migration?